### PR TITLE
Copyright year made dynamic in all the pages

### LINKED ISSUE #66: Copyright year is not dynamic

### PR DESCRIPTION
**Pull request by techrajat**
closes #66 

Copyright year made dynamic in all the pages using the following JavaScript logic: -
Screenshot (172)

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -107,3 +107,10 @@ faqQuestions.forEach((question) => {
     }
   });
 });
+
+
+if(document.querySelector('.footer-copyright')){
+  let year = new Date();
+  year = year.getFullYear();
+  document.querySelector('.footer-copyright').innerHTML = 'Copyright &#169; ' + year + ' <a href="#">Aiskcon</a>, All Right Reserved';
+}
```
Comment (AbhiPatel10) [inline js/main.js:116]: Give "/" in href

**Review by AbhiPatel10 — CHANGES_REQUESTED** [review]

**Comment by AbhiPatel10** [comment]
@techrajat Solve requested changes

**Comment by techrajat** [comment]
> @techrajat Solve requested changes

Done

**Comment by techrajat** [comment]
> > @techrajat Solve requested changes
> 
> @AbhiPatel10  Done

**Pull request merged**